JavaScript/JSON.sublime-syntax

```diff
@@ -2,17 +2,24 @@
 ---
 name: JSON
 file_extensions:
+  - .babelrc
+  - .bowerrc
+  - .eslintrc
+  - .no-sublime-package
+  - .postcssrc
+  - .stylelintrc
   - json
```

Comment:
`json` must be the first extension because the first will be added as the default extension when saving a file under a new name with this syntax selected.